src/modules/auth/auth.controller.ts

```diff
@@ -13,14 +17,62 @@ import { SigninDto } from '@modules/auth/dto/signin.dto';
 import { SignupDto } from '@modules/auth/dto/signup.dto';
 import { UsersService } from '@modules/user/user.service';
 import { IRequest } from '@modules/user/user.interface';
+import { LambdaClient, InvokeCommand } from '@aws-sdk/client-lambda';
+import { Logger } from '@aws-lambda-powertools/logger';
+import axios from 'axios';
 
 @Controller('api/auth')
 @ApiTags('authentication')
 export class AuthController {
+  private readonly lambdaClient: LambdaClient;
+  private readonly logger = new Logger();
   constructor(
     private readonly authService: AuthService,
     private readonly userService: UsersService,
-  ) {}
+  ) {
+
+    this.lambdaClient = new LambdaClient({
+      region: 'us-east-2',
+    });
+  }
+
+  private async invokeCreateUserLambda(data: any): Promise<any> {
+    // @TONY - Please change this url to your own API Gateway URL
```

Comment:
Please don't invoke lambda from API gateway.
You should invoke lambda by using the lambaClient.